spin-wagi-http/Kraftfile

```diff
@@ -3,9 +3,9 @@ spec: v0.6
 runtime: spin:latest
 
 labels:
-  cloud.unikraft.v1.instances/scale_to_zero.policy: "off"
-  cloud.unikraft.v1.instances/scale_to_zero.stateful: "false"
-  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 1000
+  cloud.unikraft.v1.instances/scale_to_zero.policy: "on"
+  cloud.unikraft.v1.instances/scale_to_zero.stateful: "true"
+  cloud.unikraft.v1.instances/scale_to_zero.cooldown_time_ms: 5000
```

Comment:
What is the reason for increasing the default cooldown time value (`1000`) to the new value (`5000`)? Is it because of a longer boot time for spin?

Reply:
I just wanted to be more lenient. I've just tried again with `1000` and it works the same. I've updated the PR.